radio/src/telemetry/frsky_sport.cpp

```diff
@@ -224,9 +224,18 @@ void sportProcessTelemetryPacketWithoutCrc(uint8_t origin, const uint8_t * packe
       }
 #endif
     }
+    else if (dataId == VALID_FRAME_RATE_ID) {
+      data = 100 - SPORT_DATA_U8(packet);
+    }
     else if (dataId == R9_PWR_ID) {
-      uint32_t r9pwr[] = {100, 200, 500, 1000};
-      data = r9pwr[SPORT_DATA_U8(packet) & 0x03];
+      // receive 'dBm' and show by 'mw'
+      uint32_t r9pwrs[][2] = {{0, 1}, {5, 3}, {10, 10}, {13, 20}, {14, 25}, {20, 100}, {23, 200}, {27, 500}, {30, 1000}};
+      uint32_t dBm = SPORT_DATA_U8(packet);
+      for (auto & r9pwr: r9pwrs)
```

Comment:
code style: open the bracket on the same line
https://github.com/opentx/opentx/wiki/OpenTX-Code-Style-Guide